src/app/Caches/Create/Features/IndexedCache.tsx

```diff
@@ -0,0 +1,150 @@
+import React, {useEffect, useState} from 'react';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Form,
+  FormGroup,
+  InputGroup,
+  Label,
+  Radio,
+  Text,
+  TextContent,
+  TextInput,
+  TextVariants,
+} from '@patternfly/react-core';
+import {global_spacer_sm} from '@patternfly/react-tokens';
+import {IndexedStorage} from "@services/infinispanRefData";
+import {useTranslation} from 'react-i18next';
+import {MoreInfoTooltip} from '@app/Common/MoreInfoTooltip';
+
+const IndexedCache = (props: {
+  indexedOptions: IndexedCache,
+  indexedOptionsModifier: (IndexedCache) => void,
+}) => {
+
+  const {t} = useTranslation();
+  const brandname = t('brandname.brandname');
+
+  //Indexed Cache
+  const [indexedStorage, setIndexedStorage] = useState<'filesystem' | 'local_heap'>(props.indexedOptions.indexedStorage);
+  const [indexedEntities, setIndexedEntities] = useState<string[]>(props.indexedOptions.indexedEntities);
+
+
+  const [entityInput, setEntityInput] = useState<string>('');
+  const [validEntity, setValidEntity] = useState<'success' | 'error' | 'default'>('default');
+
+  useEffect(() => {
+    props.indexedOptionsModifier({
+      indexedStorage: indexedStorage,
+      indexedEntities: indexedEntities,
+
+    });
+  }, [indexedStorage, indexedEntities]);
+
+  const deleteChip = (chipToDelete: string) => {
+    const newChips = indexedEntities.filter(chip => !Object.is(chip, chipToDelete));
+    setIndexedEntities(newChips);
+  };
+
+  const addChip = (newChipText: string) => {
+    setIndexedEntities([...indexedEntities, `${newChipText}`]);
+    setEntityInput('');
+  };
+
+  const helperAddEntity = () => {
+    if (entityInput.length) {
+      if (!indexedEntities.includes(entityInput)) {
+        addChip(entityInput);
+        setValidEntity('success')
+      } else {
+        setValidEntity('error');
+      }
+    }
+  }
+
+  return (
+    <Form onSubmit={(e) => {
+      e.preventDefault();
```

Comment:
@karesti Prevent default is not working, When we press enter on Indexed entities, the page get refreshed.